clients/client/src/test/java/org/projectnessie/client/http/TestHttpClient.java

```diff
@@ -417,7 +418,7 @@ public boolean equals(Object o) {
       if (this == o) {
         return true;
       }
-      if (o == null || getClass() != o.getClass()) {
+      if (!(o instanceof ExampleBean)) {
```

Comment:
imo we should try to fix all violations of a single errorprone check in a single PR (and also potentially turn the check into an error)